extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -401,9 +404,14 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       return;
     }
     this.win['dbclk_a4a_viz_change'] = true;
-    addExperimentIdToElement(
-        isExperimentOn(this.win, 'a4a-safeframe-preloading-off') ?
-        '21061136' : '21061135', this.element);
+    if (isExperimentOn(this.win, 'a4a-safeframe-preloading-off')) {
```

Comment:
You really want to use src/experiments#randomlySelectUnsetExperiments which will do this for you by passing the branches:

const sfPreloadExpName = 'a4a-safeframe-preloading-off';
 const experimentInfoMap =
        /** @type {!Object<string, !ExperimentInfo>} */ (sfPreloadExpName);
    experimentInfoMap[] = {
      isTrafficEligible: () => true,
      branches: ['21061135', '21061136'],
    };
    randomlySelectUnsetExperiments(this.win, experimentInfoMap);
  const sfPreloadExpId = getExperimentBranch(this.win, sfPreloadExpName);
if (sfPreloadExpId) {
    addExperimentIdToElement(sfPreloadExpId, this.element);
}
this.preloadSafeframe_ = sfPreloadExpId == '21061136';

Reply:
Done.